spec/quotas/quota.mdown

```diff
@@ -81,27 +83,27 @@ optimise data transfer by keeping track of changes to Quota usage separate from
 updatedProperties array may be used directly via a back-reference in a subsequent Quota/get call in the same request,
 so only these properties are returned if nothing else has changed.
 
-Servers MAY decide to add other properties to the list that they judge changing frequently.
+Servers MAY decide to add other properties to the list that they judge to be changing frequently.
 
 This method's usage is demonstrated in this document at (#requesting-latest-quota-changes).
 
 ## Quota/query
 
 This is a standard "/query" method as described in [@!RFC8620], Section 5.5.
 
-A **FilterCondition** object has the following properties, any of which may be omitted:
+A **FilterCondition** object has the following properties, any of which may be included or  omitted:
 
 * name: "String" 
   
   The Quota _name_ property contains the given string.
 
-* scopes: "Scope[]" 
+* scopes: "Scope" 
```

Comment:
Shouldn't it be 

```suggestion
* scope: "Scope" 
```

Reply:
Yep I noticed it myself while I was answering mails :) but good catch !